### PR TITLE
koordlet: fix statsinformer interface function implementation.

### PR DESCRIPTION
**Pull request by xigang**
### Ⅰ. Describe what this PR does

koordlet: fix statsinformer interface function implementation.

### Ⅱ. Does this pull request fix one issue?

```
NONE
```

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: pkg/koordlet/statesinformer/states_informer.go [hunk 1]
```diff
@@ -194,7 +194,8 @@ func (s *statesInformer) GetNode() *corev1.Node {
 	nodeInformerIf := s.states.informerPlugins[nodeInformerName]
 	nodeInformer, ok := nodeInformerIf.(*nodeInformer)
 	if !ok {
-		klog.Fatalf("node informer format error")
+		klog.Errorf("node informer format error")
+		return nil
```
Comment (jasonliu747) [inline pkg/koordlet/statesinformer/states_informer.go:198]: `return` is enough

Comment (xigang) [reply pkg/koordlet/statesinformer/states_informer.go:198]: The `GetNode() *corev1.Node` declaration in the StatesInformer interface returns' *corev1.Node ', so 'return nil' is required here.

```
type StatesInformer interface {
	GetNode() *corev1.Node
	GetNodeSLO() *slov1alpha1.NodeSLO
	GetAllPods() []*PodMeta
	GetNodeTopo() *topov1alpha1.NodeResourceTopology
}
```

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

File: pkg/koordlet/statesinformer/states_informer.go (hunk 1 above)
Comment (jasonliu747) [reply pkg/koordlet/statesinformer/states_informer.go:198]: you're right!

**Comment by xigang** [comment]
> > /cc @jasonliu747 @eahydra Please approve code.
> 
> I'll leave that decision to @zwzhang0107

/cc @zwzhang0107 Please take a look. thanks.

**Pull request merged**